make/config.mk

```diff
@@ -125,16 +125,6 @@ USE_MKLDNN = 0
 USE_NGRAPH = 0
 USE_NGRAPH_IE = 0
 USE_NGRAPH_DISTRIBUTED = 0
-ADD_NGRAPH_LIBDIR_TO_MXNET_RPATH=1
-override NGRAPH_EXTRA_CMAKE_FLAGS += -DNGRAPH_UNIT_TEST_ENABLE=0 -DNGRAPH_TOOLS_ENABLE=0
```

Comment:
How does users pass this if needed.

Reply:
Haven't really solved that problem yet...